### ISSUE TITLE
[New Core]: Smash TV

### ISSUE THREAD

**Issue by atrac17** [openfpga-cores-inventory/analogue-pocket#812]
### Core Author username

Coin-OpCollection

### Core .yml snippet

```yml
display_name: Smash TV
repository: Distribution-OpenFPGA
filter: pram0d.yunit_
```

**Comment by atrac17** [comment]
This is a multi-game core, additional release will fall under filter.

**Comment by joshcampbell191** [comment]
Closed per #813.
